google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/AppEngineStandardFacetEditor.java

```diff
@@ -167,50 +168,42 @@ private boolean isManagedLibrariesEnabled() {
 
     @Override
     public void afterLibraryAdded(final Library addedLibrary) {
-      AppEngineStandardMavenLibrary library = AppEngineStandardMavenLibrary
-          .getLibraryByMavenDisplayName(addedLibrary.getName());
+      Optional<AppEngineStandardMavenLibrary> libraryOptional =
+          AppEngineStandardMavenLibrary.getLibraryByMavenDisplayName(addedLibrary.getName());
 
-      if (library == null) {
-        return;
-      }
+      libraryOptional.ifPresent(
+          library -> {
+            final DependencyScope scope = library.getScope();
 
-      final DependencyScope scope = library.getScope();
+            ApplicationManager.getApplication().runWriteAction(() -> {
```

Comment:
I created new projects with all Maven dependencies checked, and verified that all of the dependencies were there.

Since you're more familiar with this code, do you know of other useful tests I can run?